beams/beams/doctype/local_enquiry_report/local_enquiry_report.py

```diff
@@ -24,18 +36,45 @@ def information_required(self):
                 self.information_collected_by = frappe.session.user
 
             missing_fields = []
-
-            if not self.information_given_by:
-                missing_fields.append("Information given by")
-            if not self.information_given_by_designation:
-                missing_fields.append("Information given by Designation")
+            # Ensure enquiry officer selection based on workflow state.
+            if self.workflow_state == 'Assigned to Enquiry Officer':
+                if not self.enquiry_officer:
+                    frappe.msgprint(_("Please select an enquiry officer before proceeding."))
+                    # Optionally, raise an exception to prevent saving the document if no officer is selected
+                    raise frappe.ValidationError(_("Enquiry officer must be selected when the state is 'Assigned to Admin'."))
+                else:
+                    # If the enquiry officer is selected, update the field
+                    frappe.db.set_value("Local Enquiry Report", self.name, "enquiry_officer", self.enquiry_officer)
```

Comment:
What is this line supposed to do? this line is simmply setting the same value in the same field